tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Summary.java

```diff
@@ -39,8 +42,18 @@
 @Tag(name = "voting", description = "APIs for voting")
 public class Summary {
 
+    private static final int AUTO_SNAP_HOUR_OF_DAY = 1; // 1 am
+    private static final String AUTO_SNAP_TIME_ZONE = "America/Los_Angeles";
```

Comment:
the server time zone seems less relevant than Unicode being a California corporation -- the rationale is to avoid making the (resource-intensive) snapshots during our busiest working hours -- though of course some of us might be working anywhere in the world or any time of day or night

anyway if the server were to move to another time zone, for whatever reason, there's not really a reason to change the snapshot timing accordingly

Reply:
fine… probably should be a config in case we need this for something else also.  Fine for now.

Reply:
sounds good